dissect/target/plugins/apps/av/mcafee.py

```diff
@@ -0,0 +1,120 @@
+import ipaddress
+import re
+from collections import defaultdict
+from pathlib import Path
+from typing import Iterator
+
+from dissect.sql import SQLite3
+from dissect.util.ts import from_unix
+
+from dissect.target.exceptions import UnsupportedPluginError
+from dissect.target.helpers.record import TargetRecordDescriptor
+from dissect.target.plugin import Plugin, export
+
+McAfeeMscLogRecord = TargetRecordDescriptor(
+    "application/av/mcafee/msc",
```

Comment:
Would it be useful to also split the records in firewall and virus specific versions? As most of the useful data is mutually exclusive anyway?

Reply:
Agreed, this now evolves into separate records.